compiler-core/src/error.rs

```diff
@@ -1335,9 +1348,13 @@ Names in a Gleam module must be unique so one will need to be renamed."
                             },
                             path: path.clone(),
                             src: src.clone(),
-                            extra_labels: vec![Label {
-                                text: Some("First imported here".into()),
-                                span: *previous_location,
+                            extra_labels: vec![ExtraLocation {
```

Comment:
Perhaps this name as it's not always a location
```suggestion
                            extra_labels: vec![ExtraLabel {
```

Reply:
works for me